src/sider/manglendemedvirkning/ManglendeMedvirkning.tsx

```diff
@@ -1,31 +1,20 @@
-import React from "react";
-import ForhandsvarselSkjema from "@/sider/manglendemedvirkning/forhandsvarsel/ForhandsvarselSkjema";
+import React, { useState } from "react";
 import { useManglendeMedvirkningVurderingQuery } from "@/data/manglendemedvirkning/manglendeMedvirkningQueryHooks";
 import { VurderingType } from "@/data/manglendemedvirkning/manglendeMedvirkningTypes";
-import { Alert, Box } from "@navikt/ds-react";
-import ForhandsvarselSendt from "@/sider/manglendemedvirkning/forhandsvarsel/ForhandsvarselSendt";
-import { useNotification } from "@/context/notification/NotificationContext";
+import { ManglendeMedvirkningStartetVurdering } from "@/sider/manglendemedvirkning/ManglendeMedvirkningStartetVurdering";
+import { ManglendeMedvirkningNyVurdering } from "@/sider/manglendemedvirkning/ManglendeMedvirkningNyVurdering";
 
 export default function ManglendeMedvirkning() {
-  const { notification } = useNotification();
   const { sisteVurdering } = useManglendeMedvirkningVurderingQuery();
   const isForhandsvarsel =
     sisteVurdering?.vurderingType === VurderingType.FORHANDSVARSEL;
+  const [showStartetVurdering, setShowStartetVurdering] = useState(false);
 
-  return (
-    <>
-      {notification && (
-        <Alert variant="success" className="mb-2">
-          {notification.message}
-        </Alert>
-      )}
-      <Box>
-        {isForhandsvarsel ? (
-          <ForhandsvarselSendt forhandsvarsel={sisteVurdering} />
-        ) : (
-          <ForhandsvarselSkjema />
-        )}
-      </Box>
-    </>
+  return showStartetVurdering || isForhandsvarsel ? (
+    <ManglendeMedvirkningStartetVurdering sisteVurdering={sisteVurdering} />
+  ) : (
+    <ManglendeMedvirkningNyVurdering
+      handleClick={() => setShowStartetVurdering(true)}
+    />
```

Comment:
Vil vi ikke at den skal defaulte til forhåndsvarsel-siden? Det er det vi gjør i 8-4.

Reply:
Tror dette skal være tilsvarende 8-4 hvor vi viser "Ingen vurderinger har blitt gjort, trykk på 'Start ny vurdering' for å sende forhåndsvarsel" og så ny vurdering-knapp dersom det ikke finnes noen vurderinger.

Reply:
Ja, jaggu 
<img width="956" alt="image" src="https://github.com/user-attachments/assets/497d2dbd-d15d-47be-a74f-5841aa655f33">